tests/unit/peapods/test_zmq.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from jina.peapods.zmq import _extract_bytes_from_msg
+from jina.proto import jina_pb2
+from jina.drivers.helper import array2pb, pb2array
+
+random_np_array = np.random.randint(10, size=(50, 10))
```

Comment:
Why defining these outside the test? The only reason I see, it that multiple tests use them. Anyhow, there is only one test.

Nevertheless, if they are outside, I'd prefer them being constants, e.g. `RANDOM_NP_ARRAY`, `BUFFER` and `TEXT`